src/PKSim.Core/Snapshots/Mappers/ParameterContainerSnapshotMapperBase.cs

```diff
@@ -38,16 +37,10 @@ protected override Task<TSnapshot> SnapshotFrom(TModel model, Action<TSnapshot>
 
       protected virtual Task AddModelParametersToSnapshot(TModel model, TSnapshot snapshot)
       {
-         return AddParametersToSnapshot(model.AllParameters(ParameterHasChanged), snapshot);
-      }
-
-      protected virtual bool ParameterHasChanged(IParameter parameter)
```

Comment:
I have moved this method as an extension method. This will probably be merged with the ValueDiffersFromDefault when we are sure of how we will implement this method